stripe/src/main/java/com/stripe/android/view/StripeEditText.kt

```diff
@@ -257,4 +262,8 @@ open class StripeEditText @JvmOverloads constructor(
             return super.deleteSurroundingText(beforeLength, afterLength)
         }
     }
+
+    private companion object {
+        private const val COMMON_HINT_DELAY = 90L
```

Comment:
[optional] I find it useful to name time constants with their unit, eg `COMMON_HINT_DELAY_MILLIS`